cmd/flowserver/flow.go

```diff
@@ -37,9 +32,9 @@ func (r *flowResponse) MarshalJSON() ([]byte, error) {
 
 type startRequest struct {
 	Environment *json.RawMessage       `json:"environment"`
-	Flows       json.RawMessage        `json:"flows"    validate:"required"`
-	Channels    []json.RawMessage      `json:"channels,omit_empty"`
-	Contact     json.RawMessage        `json:"contact"  validate:"required"`
+	Assets      json.RawMessage        `json:"assets"           validate:"required"`
+	Flow        flows.FlowUUID         `json:"flow_uuid"        validate:"required"`
```

Comment:
Why the change from this being the first? It being the first has the nice property that it removes a whole class of possible errors.